app/_locales/pt_BR/messages.json

```diff
@@ -3,105 +3,180 @@
+  },
+  "affirmAgree": {
+    "message": "Concordo"
+  },
+  "aggregatorFeeCost": {
+    "message": "Taxa de rede do agregador"
+  },
+  "alertDisableTooltip": {
+    "message": "Isso pode ser alterado em \"Configurações > Alertas\""
+  },
+  "alertSettingsUnconnectedAccount": {
+    "message": "Navegar por um website com uma conta não conectada selecionada"
+  },
+  "alertSettingsUnconnectedAccountDescription": {
+    "message": "Este alerta é exibido no pop-up quando você estiver navegando por um site conectado da Web3, mas a conta selecionada no momento não estiver conectada."
+  },
+  "alerts": {
+    "message": "Alertas"
+  },
+  "alertsSettingsDescription": {
+    "message": "Habilitar ou desabilitar cada alerta"
+  },
+  "allowExternalExtensionTo": {
+    "message": "Permitir que esta extensão externa:"
+  },
+  "allowOriginSpendToken": {
+    "message": "Permitir que $1 gaste seus(suas) $2?",
+    "description": "$1 is the url of the site and $2 is the symbol of the token they are requesting to spend"
+  },
+  "allowThisSiteTo": {
+    "message": "Permitir que este site:"
+  },
+  "allowWithdrawAndSpend": {
+    "message": "Permitir que $1 saque e gaste do seguinte valor:",
+    "description": "The url of the site that requested permission to 'withdraw and spend'"
   },
   "amount": {
     "message": "Valor"
   },
+  "amountInEth": {
+    "message": "$1 ETH",
+    "description": "Displays an eth amount to the user. $1 is a decimal number"
+  },
+  "amountWithColon": {
+    "message": "Valor:"
+  },
   "appDescription": {
-    "message": "Uma Carteira Ethereum no seu Navegador",
+    "message": "Uma carteira do Ethereum no seu navegador",
     "description": "The description of the application"
   },
   "appName": {
     "message": "MetaMask",
     "description": "The name of the application"
   },
+  "approvalAndAggregatorTxFeeCost": {
+    "message": "Aprovação e taxa da rede do agregador"
+  },
+  "approvalTxGasCost": {
+    "message": "Custo em gas da transação de aprovação"
+  },
   "approve": {
-    "message": "Aprovar"
+    "message": "Aprovar limite de gastos"
+  },
+  "approveSpendLimit": {
+    "message": "Aprovar limite de gastos de $1",
+    "description": "The token symbol that is being approved"
   },
   "approved": {
     "message": "Aprovado"
   },
   "asset": {
     "message": "Ativo"
   },
+  "assets": {
+    "message": "Ativos"
+  },
   "attemptToCancel": {
-    "message": "Tentar Cancelar?"
+    "message": "Tentar cancelar?"
   },
   "attemptToCancelDescription": {
-    "message": "Enviar esta tentativa não garante que sua transação original será cancelada. Se a tentativa de cancelamento for bem-sucedida, será feita uma cobrança pela taxa de transação acima."
+    "message": "Enviar esta tentativa não garante que a transação original será cancelada. Se a tentativa de cancelamento tiver sucesso, será cobrada a taxa de transação acima."
```

Comment:
This isn't wrong, it just doesn't read naturally. I can't explain why.

Change the sentence to:

```
Se a tentativa de cancelamento tiver sucesso, a taxa de transação acima será cobrada.
```

Reply:
:+1: 
```suggestion
    "message": "Enviar esta tentativa não garante que a transação original será cancelada. Se a tentativa de cancelamento tiver sucesso, a taxa de transação acima será cobrada."
```